addons/mail/static/src/core/common/thread_model.js

```diff
@@ -1046,17 +1046,20 @@ export class Thread extends Record {
             if (parentId) {
                 tmpData.parentMessage = this.store.Message.get(parentId);
             }
-            const prettyContent = await prettifyMessageContent(
+            let prettyBody = await prettifyMessageContent(
                 body,
                 this.store.getMentionsFromText(body, {
                     mentionedChannels,
                     mentionedPartners,
                 })
             );
+            if (this.model === "discuss.channel") {
+                prettyBody = `<odoo-markdown>${prettyBody}</odoo-markdown>`;
+            }
```

Comment:
I'm confused why do we add `<odoo-markdown>` in several different places. It's done in `getMessagePostParams` which is called here above.